README.md

```diff
@@ -1,4 +1,7 @@
 # Chameleon
+[![Build Status](https://travis-ci.org/agamdua/chameleon.svg?branch=master)](https://travis-ci.org/agamdua/chameleon)
```

Comment:
Go to my master check out the button it's really cool

What you can do is login to Travis - it's free for open source projects - and enable this project to build for you. It is then a PR builder and can be configured as a push builder as well!

 It's dead simple once you have the yaml file in place.